docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -151,42 +151,24 @@ Teleport will respond back with a discovery info:
 ```bash
 $ curl https://tele.example.com/webapi/ping
 
-{"server_version":"5.0.0-dev","min_client_version":"3.0.0"}
+{"server_version":"6.0.0","min_client_version":"3.0.0"}
 ```
 
 ## Step 2/3 Create a local admin
 
 Local users are a reliable fallback for cases when the SSO provider is down.
-Let's create a local admin `alice` who has access to Kubernetes group `system:masters`.
-
-Save this role as `member.yaml`:
-
-```yaml
-kind: role
-version: v3
-metadata:
-  name: member
-spec:
-  allow:
-    # This field is used for SSH logins. You have to keep 'logins' as a non-empty random value
-    # for Kubernetes to work until we fix it.
-    logins: ['keep-this-value-here']
-    kubernetes_groups: ["system:masters"]
-```
+Let's create a local admin `teleport-admin` who has access to Kubernetes group `system:masters`.
```

Comment:
Why did you remove this example?

Reply:
I removed it to use the preset role instead `kubectl exec -ti ${POD?} -- tctl users add teleport-admin --roles=editor,access --kubernetes_groups=system:masters --logins=k8s-keep-this-login` and to require one less YAML file for setup.

Reply:
( reverted this in the latest commit )